### PR TITLE
Adding ppc64le architecture support on travis-ci

### PR DESCRIPTION
**Pull request by kishorkunal-raj**
Hi,
I had added ppc64le architecture support on travis-ci and looks like its been successfully added. Also I had added installation of ninja and glib for uniformity across the platforms. I believe it is ready for the final review and merge.

Please have a look.

Thanks!!
Kishor Kunal Raj

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,15 +27,17 @@ addons:
             - elinks
             - libgcrypt11-dev
             - libotr5-dev
+            - ibglib2.0-dev
```
Comment (dequis) [inline .travis.yml:30]: Wow, what, how does this not fail?

**Review by dequis — COMMENTED** [review]
This downgrades ninja from 1.9.0 to 1.5.1 and glib from 2.58.3 to 2.48.2. Not really sure why we were using those specifically though, @ailin-nemui?

**Comment by ailin-nemui** [comment]
I considered bumping our version requirement to 2.58 in the past but we have compat code in place now.

Meson 0.54 will require Ninja newer than 1.7, and compdb requires Ninja 1.9, but I guess we are not using those on Travis

We probably should not leave behind commented-out code or references to now non-existing folders if we decide to use system-provided ninja and glib

**Comment by ailin-nemui** [comment]
is there any point in testing Irssi compilation on ppc64le? is there anything in particular where our code might fail in the future?

**Comment by gerrith3** [comment]
This code appears to be written in C/C++ - with a different compiler, libraries, possible differences in distro dependencies, there are a lot of reasons why this *could* fail, testing will ensure no regresssion, which will be really helpful when rebuilding for the next distro release.

**Comment by kishorkunal-raj** [comment]
Hi @ailin-nemui , 
Kindly let us know your opinion towards the changes

Thanks!

**Comment by ailin-nemui** [comment]
you can fix the obvious issues and address my requests from above comment first

**Pull request closed without merging**